ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivCreatePrivacyGroupTest.java

```diff
@@ -50,7 +50,7 @@
   @Before
   public void setUp() {
     when(failingEnclave.createPrivacyGroup(any(CreatePrivacyGroupRequest.class)))
-        .thenThrow(new EnclaveException(""));
+        .thenThrow(new EnclaveClientException(500, ""));
```

Comment:
I believe EnclaveClientException should have a status code < 500. Even though it might not change the outcome of the test. I think it is good to be consistent :)